commons/src/config/mod.rs

```diff
@@ -117,23 +118,28 @@ where
     Ok(HashSet::from_iter(entries?))
 }
 
-pub fn parse_to_locale_pathbuf_hashset(s: &str) -> Result<HashMap<Locale, PathBuf>, clap::Error> {
+fn parse_locale_path_buf(s: &str) -> Result<(Locale, PathBuf), Box<dyn Error + Send + Sync + 'static>>
```

Comment:
I think it might be better to return `clap::Error` instead of boxing.
Probably using [`raw()`](https://docs.rs/clap/latest/clap/error/struct.Error.html#tymethod.raw).

This way, you can return an error instead of default `invalid path`.